pkg/apiserver/apiserver.go

```diff
@@ -100,6 +101,7 @@ func (s *Service) Start(ctx context.Context) error {
 	s.app = fx.New(
 		fx.Logger(utils.NewFxPrinter()),
 		fx.Provide(
+			featureflag.ProvideRegistry(s.config.FeatureVersion),
```

Comment:
How about:

```go
fx.Supply(featureflag.NewRegistry(s.config.FeatureVersion))
```

Reply:
Upgrade fx to 1.16.0 to support `fx.Supply`.